### ISSUE TITLE
Add Phenotype All Species search (exists in GN1)

### ISSUE THREAD

**Issue by zsloan** [genenetwork/genenetwork2#16]
_From @zsloan on May 5, 2015 21:49_

The All Species search is not yet implemented, but Lei's global search code should be able to take its place once it is finished.

_Copied from original issue: zsloan/genenetwork2#14_

**Comment by lyan6** [comment]
Merged into this issue:

https://github.com/genenetwork/genenetwork2/issues/31
